tools/cli/internal/changelog/outputfilter/squash.go

```diff
@@ -18,29 +18,94 @@ type SquashHandler interface {
 	Squash(entriesGroupedByOperationID map[string][]*OasDiffEntry) []*OasDiffEntry
 }
 
-func newSquashHandlers() map[string]func(map[string][]*OasDiffEntry) ([]*OasDiffEntry, error) {
-	return map[string]func(map[string][]*OasDiffEntry) ([]*OasDiffEntry, error){
+type handler struct {
+	id       string
+	squasher func(map[string][]*OasDiffEntry) ([]*OasDiffEntry, error)
+}
+
+func newSquashHandlers() []handler {
```

Comment:
we want to keep the order of the changes when the print the changelog to match the ones in evergreen